### PR TITLE
Fix Compatibility Test CI

### PR DESCRIPTION
**Pull request by chatton**
## Description

closes: #4690

Compatibility tests:

https://github.com/cosmos/ibc-go/actions/runs/6224433627

good news

This run is just transfer, ~will run a larger set when we don't need to worry about hogging all the runners~

We can do a full run once this is merged.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: e2e/testsuite/testconfig.go [hunk 1]
```diff
@@ -55,9 +55,9 @@ const (
 	defaultBinary = "simd"
 	// defaultRlyTag is the tag that will be used if no relayer tag is specified.
 	// all images are here https://github.com/cosmos/relayer/pkgs/container/relayer/versions
-	defaultRlyTag = "latest" // "andrew-tendermint_v0.37" // "v2.2.0"
+	defaultRlyTag = "latest"
 	// defaultHermesTag is the tag that will be used if no relayer tag is specified for hermes.
-	defaultHermesTag = "v1.6.0"
+	defaultHermesTag = "bef2f53"
```
Comment (chatton) [inline e2e/testsuite/testconfig.go:60]: this value used to be picked up by the config file.

**Review by damiannolan — APPROVED** [review]
Nice! Thank you and lgtm! 🙏

**Review by DimitrisJim — APPROVED** [review]

**Review by colin-axner — APPROVED** [review]
Wahooooo! You are the best @chatton :rocket: love the pr image :smile:

**Review by charleenfei — APPROVED** [review]
thank you!!

**Pull request merged**